Bulldozer.CSV/Maps/Attributes.cs

```diff
@@ -138,11 +138,12 @@ private int LoadEntityAttributeValues()
 
             var definedTypeDict = DefinedTypeCache.All().ToDictionary( k => k.Id, v => v );
             var attributeValuesByEntityType = this.EntityAttributeValueCsvList
+                                                .DistinctBy( av => new { av.AttributeKey, av.EntityId } )  // Protect against duplicates in import data
```

Comment:
AttributeKey could represent more than one attribute based on entity type of attribute. Probably want to back this change back out.